terragraph_planner/los/test/test_building_group.py

```diff
@@ -45,10 +45,10 @@ def test_preprocess(self) -> None:
             )
         )
         coords2 = [
-            (0.001, 0.0009),
             (0.0009, 0.0009),
             (0.0009, 0.001),
             (0.001, 0.001),
+            (0.001, 0.0009),
```

Comment:
I'm a little concerned that 3rd party package versions can impact our unit test results. Do you know what caused this change? Do we have any safeguards for this (e.g., nightly unit testing that would warn us if a package update breaks some unit tests)?

Reply:
I guess the shapely library is updated somehow so the order of coords of the result of `geometry.intersection` is changed. The result is still good, but the coordinate order is changed.

Reply:
It means that without nightly testing a unit test could fail without having anything to do with the PR because a third party library got updated.

Can we add a nightly testing GitHub action?

Reply:
Sorry, I don't know what "nightly testing" means here. Isn't that a test automatically run every day/night?

Reply:
Yes, independent of a PR. It means that if something out of our control changes that breaks a test, we find out immediately. So, if a third party library changes something that breaks our tests, we know that within 24 hours rather than waiting for the next PR to find out.